union-ai-admin/gcp/union-ai-admin-role.yaml

```diff
@@ -1,6 +1,21 @@
 title: Unionai Administrator
 stage: GA
 includedPermissions:
+- artifactregistry.repositories.create
```

Comment:
I snipped this out of Artfacit Registry Admin role. There are likely a few line entries we can snip but I am curious if it is worth it or retain the permissions as is.